src/docs/product/integrations/integration-platform/ui-components/issue-link.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: Issue Link
+sidebar_order: 1
+description: "Learn about the issue link UI component."
+---
+
+With an issue linking UI component, you can connect Sentry issues with a task in your project management system. This functionality provides a way to use any project management tool you use or develop.
+
+The issue link component displays with the text "Link &lt;Service&gt; Issue" in the **Issue Details** sidebar:
+
+![Issue linking sidebar showing Shortcut icon.](issue-sidebar-icons.png)
+
+Clicking the link opens a modal allowing the user to create or link the issue to a task in the external service:
+![Sentry modal that creates a Shortcut Story that's linked to a Sentry issue.](../../issue-tracking/shortcut/shortcut_create_story_with_issue.png)
+
+## Issue Link Schema
+
+```json {filename:schema.json}
+{
+  "elements": [
+    {
+      "type": "issue-link",
+      "link": {
+        "uri": <String>,
+        "required_fields": <Array<FormField>>,
+        "optional_fields": <Array<FormField>>
+      },
+      "create": {
+        // Same as "link" schema
+      }
+    }
+  ]
+}
+```
+
+### Attributes of the Link and Create Fields
+
+- `uri` - (Required) The URI to request when the User submits the Link/Create Issue form.
+- `required_fields` - (Required) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User is required to complete.
+- `optional_fields` - (Optional) List of [FormField](/product/integrations/integration-platform/ui-components/formfield) components the User may complete.
+
+
+## Issue Link Request and Response
+
+When a user attempts to create or link an issue, we will send a request to your service based on the provided `uri`.
+
+### Request Format
+
+We send a `POST` request, so all the information is stored in the body.
```

Comment:
I think these two sentences can be written one after another and a heading removed so that we just have `## Request Format` as an h2. I'm just judging by the sidebar, and it feels better like that.